<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Kantor C++</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pageLogowanie">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>364</width>
        <height>269</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="labelLogin">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Login:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelHaslo">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Hasło:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButtonZaloguj">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Zaloguj</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditLogin">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEditHaslo">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelLogowanie">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>36</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Logowanie</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>labelLogin</zorder>
      <zorder>labelHaslo</zorder>
      <zorder>lineEditHaslo</zorder>
      <zorder>lineEditLogin</zorder>
      <zorder>pushButtonZaloguj</zorder>
      <zorder>labelLogowanie</zorder>
     </widget>
     <widget class="QLabel" name="labelError">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>420</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Niepoprawny login lub hasło.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageProfil">
     <widget class="QLabel" name="labelKantor">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Kantor</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPowitanie">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Witaj ponownie, </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonWyloguj">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wyloguj</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>170</y>
        <width>181</width>
        <height>315</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="1">
        <widget class="QLabel" name="labelJPYValue">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="labelGBPValue">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelCHF">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CHF:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelUSD">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>USD:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelEUR">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>EUR:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelGBP">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>GBP:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelCZK">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CZK:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelPLNValue">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="labelEURValue">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPLN">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PLN:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="labelUSDValue">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="labelCHFValue">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelJPY">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>JPY:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="labelCADValue">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="labelCZKValue">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelCAD">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CAD:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="labelPortfel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>146</width>
        <height>34</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Stan portfela:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelWymiana">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wymiana walut</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxWalutaL">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>250</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxWalutaR">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>250</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBoxKwota">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>250</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonWymien">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>300</y>
        <width>180</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wymień</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonWykresy">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>400</y>
        <width>260</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wykresy kursów walut</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageWykresy">
     <widget class="QPushButton" name="pushButtonPowrot">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>20</y>
        <width>150</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Powrót</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxWaluta">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>475</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRysuj">
      <property name="geometry">
       <rect>
        <x>425</x>
        <y>475</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rysuj</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsViewWykres">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>600</width>
        <height>350</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
